ports/blas/CMakeLists.txt

```
@@ -1,4 +1,4 @@
 cmake_minimum_required(VERSION 3.17)
 
-project(Find_BLAS_external)
+project(Find_BLAS_external LANGUAGES CXX)

```

```suggestion
project(Find_BLAS_external)
```

unnecessary change. Will be removed after team review.